frontend/src/main/java/com/psephoi/mvc/manager/PollManagerImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.psephoi.mvc.manager;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.psephoi.mvc.model.Poll;
+
+@Service
+public class PollManagerImpl implements PollManager{
+	
+	private final String restURL = "http://localhost:8080/";

```

Comment:
ideally you shouldn't ever hardcode the protocol and host of the provider, but in this context it's ok.